comps/llms/text-generation/ollama/llm.py

```diff
@@ -21,7 +21,7 @@
 def llm_generate(input: LLMParamsDoc):
     ollama = Ollama(
```

Comment:
does the Ollama initialization have latency gap?

Reply:
Good point, we will cover the test.

Reply:
Yes, I verify and it sometimes has latency gap. Here is the default Ollama params https://github.com/ollama/ollama/blob/main/docs/modelfile.md

But I think it is safe to just use OpenAI default ones.

Reply:
Also I guess this model name should be passed as input param. WIll adapt https://github.com/opea-project/GenAIComps/pull/309 later.